.github/workflows/ci.yml

```diff
@@ -19,20 +19,21 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v1
 
-      - name: Setup dotnet
-        uses: actions/setup-dotnet@v1
+      - name: Setup Node 14
+        uses: actions/setup-node@v2
         with:
-          dotnet-version: 3.1.101
+          node-version: '14'
 
-      - name: Build backend
-        shell: bash
+      - name: Setup Server dependencies
+        working-directory: ./Server
         run: |
-          dotnet build ./Chat
+          npm install
 
-      - name: Test backend
+      - name: Build backend
+        working-directory: ./Server
         shell: bash
         run: |
-          dotnet test ./Chat
+          npm run build
 
   clean_frontend:
```

Comment:
nit: rename these actions
the previous: `build_and_test` -> `build_and_test_server` (and update name to follow `Build and test server`)
this one to something like: `build_and_test_frontend` and `Build and test web app`